fastf1/core.py

```diff
@@ -894,6 +894,8 @@ def calculate_driver_ahead(self):
         for drv in self.session.drivers:
             # find correct first relevant lap; very important for correct zero point in distance
             drv_laps = self.session.laps[self.session.laps['DriverNumber'] == drv]
+            if not drv_laps.empty: # Only include drivers who participated in this session
```

Comment:
There's one single space missing before the inline comment. There need to be two spaces... sorry

Reply:
No problem, I've just added it.

Reply:
I just saw this now when I wanted to merge it, the logic is wrong, The `not` shouldn't be there if we use continue.

Reply:
Oops my bad...fixed now